### PR TITLE
225 all image storage

### PR DESCRIPTION
**Pull request by tbellicha**
First part of my branch. I need to upload dynamically (from the Website) images

- [x] Change IP to the new VPS
- [x] Dynamic images in:
  - [x] Articles list
  - [x] Article's details
  - [x] Favorites list
  - [x] Articles list from the BottomSheet on the map
  - [x] Rentals list
  - [x] Rent details
  - [x] Return a rent
- [x] Carousel slider in the article's details
- [x] Rent fixture for the admin account
- [x] Update server packages

image
[Screen_recording_20240813_222637.webm](https://github.com/user-attachments/assets/c47267fc-ec07-419f-ad34-5e0588f1f1f2)

### CODE REVIEW

**Review by hugolopespinto — APPROVED** [review]
Good job !

File: Server/controllers/Common/images.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const { getSignedUrl } = require("@aws-sdk/s3-request-presigner");
+const { GetObjectCommand, ListObjectsV2Command } = require("@aws-sdk/client-s3");
+
+const imagesMiddleware = require('../../middleware/images');
+const itemsCtrl = require('../../controllers/Common/items');
+
+const listFilesInS3Folder = async (folderPath) => {
```
Comment (31Nathan) [inline Server/controllers/Common/images.js:7]: Missing doc

File: Server/package-lock.json
Comment (31Nathan) [inline Server/package-lock.json:1]: To not push

File: Server/routes/Web/images.js
Comment (31Nathan) [inline Server/routes/Web/images.js:1]: Route documentation missing

**Review by 31Nathan — COMMENTED** [review]
Great Job ! Small changes and documentation please

**Review by CedricCORGE — APPROVED** [review]

**Pull request merged**